scala3/index.md

```diff
@@ -22,6 +22,10 @@ sections:
         description: "An online book introducing the main language features."
         icon: "fa fa-book"
         link: /scala3/book/introduction.html
+      - title: "Scaladoc new features"
+        description: "Highlights of new features for Scaladoc"
+        icon: "fa fa-star"
+        link: /scala3/scaladoc.html
```

Comment:
This list (which ideally has three elements :) ) here should be the things newcomers should look at. Maybe we can move Scaladoc to the section "More Detailed Information"?

Reply:
Moved